metabase/metabase
# Add more missing translations

### Description

Add the missing translation reported in https://github.com/metabase/metabase/issues/33046.

Please note that from the original report
> 1. enterprise/frontend/src/metabase-enterprise/audit_app/lib/cards (in each file in each object the name field without t...)

Actually, not all `name` attributes should be translated. I only include the translations that are visible in our app.

### How to verify

Go to all places I've shown in the inline screenshots. For certain strings that are already translated, you'll see those strings being translated when changing language in the user settings. While there are some strings that need translation.

### Checklist

- [ ] ~Tests have been added/updated to cover changes in this PR~ No test since this is just adding missing translation
